Integrations/integration-Kenna.yml

```diff
@@ -0,0 +1,487 @@
+                    'Name' : res[i].cve_id
+                };
+
+                context.Kenna[i]={};
+                context.Kenna[i].AssetID = res[i].asset_id;
+                context.Kenna[i].Connectors = [];
+                for (var j = 0; j < res[i].connectors.length; j++) {
+                    context.Kenna[i].Connectors[j] = {};
+                    context.Kenna[i].Connectors[j].DefinitionName = res[i].connectors[j].connector_definition_name;
+                    context.Kenna[i].Connectors[j].ID = res[i].connectors[j].id;
+                    context.Kenna[i].Connectors[j].Name = res[i].connectors[j].name;
+                    context.Kenna[i].Connectors[j].Vendor = res[i].connectors[j].vendor;
+                }
+                context.Kenna[i].CveID = res[i].cve_id;
+                context.Kenna[i].FixID = res[i].fix_id;
+                context.Kenna[i].ID = res[i].id;
+                context.Kenna[i].Patch = res[i].patch;
+                context.Kenna[i].RiskMeterScore = res[i].risk_meter_score;
+                context.Kenna[i].ScannerVulnerabilities = [];
+                for (var j = 0; j < res[i].scanner_vulnerabilities.length; j++) {
+                    context.Kenna[i].ScannerVulnerabilities[j] = {};
+                    context.Kenna[i].ScannerVulnerabilities[j].ExternalID = res[i].scanner_vulnerabilities[j].external_unique_id;
+                    context.Kenna[i].ScannerVulnerabilities[j].Open = res[i].scanner_vulnerabilities[j].open;
+                    context.Kenna[i].ScannerVulnerabilities[j].Port = res[i].scanner_vulnerabilities[j].port;
+                }
+                context.Kenna[i].Score = res[i].score;
+                if (res[i].service_ticket) {
+                    context.Kenna[i].ServiceTicket = {};
+                    context.Kenna[i].ServiceTicket.DueDate = res[i].service_ticket.due_date;
+                    context.Kenna[i].ServiceTicket.ExternalIdentifier = res[i].service_ticket.external_identifier;
+                    context.Kenna[i].ServiceTicket.Status = res[i].service_ticket.status;
+                    context.Kenna[i].ServiceTicket.TicketType = res[i].service_ticket.ticket_type;
+                }
+                context.Kenna[i].Severity = res[i].severity;
+                context.Kenna[i].Status = res[i].status;
+                context.Kenna[i].Threat = res[i].threat;
+                context.Kenna[i].TopPriority = res[i].top_priority;
+            }
+
+            return {
+                Type: entryTypes.note,
+                Contents: res,
+                ContentsFormat: formats.json,
+                ReadableContentsFormat : formats.markdown,
+                EntryContext : context,
+                HumanReadable: tableToMarkdown('Kenna Vulnerabilities', hr)
+            };
+    }
+
+    function searchVulnerabilitiesRequest(urlQuery) {
+        var requestUrl = url.replace(/[\/]+$/, '') + '/vulnerabilities/search' + encodeToURLQuery(urlQuery);
```

Comment:
the replace you should do at the top, like this `var URL = params.url.replace(/[\/]+$/, '')`